.github/workflows/end2end.yaml

```diff
@@ -557,17 +500,12 @@ jobs:
       - name: Clean Up
         run: kind delete cluster
 
-  end2end-https:
+  end2end-2-shards-http:
```

Comment:
should we not run these tests on both 1 and 2-shards, to have a good comparison point?
i.e. keep a single `end2end-http(s)` job and add a matrix parameter for the number of shard/topology

Reply:
I'm not sure it's useful, the diff between http and https is the activation of the https, and otherwise, all these tests are covered by CTST today, so with https enabled.